### ISSUE TITLE
wttr.in Telegram Bot

### ISSUE THREAD

**Issue by MartinHennrich** [chubin/wttr.in#253]
Do you know if there is already a [Telegram](https://telegram.org/) [bot](https://core.telegram.org/api#bot-api) that uses wttr.in?

**Comment by Asternitix** [comment]
probably, check Telegram's marketplace or whatever they have. Either way, get site previews working and wttr.in will put the forecast in the preview. That's how I put forecasts in chats; not sure if it works in Telegram, but that's how it could be done.

**Comment by chubin** [comment]
Yes, there are several unofficial telegram bots that use wttr.in.
For example:

* https://github.com/vlcty/WttrInTelegramBot by @victy (the repository is archived; I believe, that the reason for that was that the author, Josef Stautner, moved to Gitlab; anyway, you can still use the code);

* https://github.com/CptPie/telegram-shipit-bot This bot uses wttr.in as source for weather information (when using the `/weather` command)

@MartinHennrich Martian, do you want to create one?
